src/services/test-client-config.ts.example

```diff
@@ -1,4 +1,4 @@
-import { ClientConfig } from "../serviceModels/clientConfig";
```

Comment:
Because this file name changed, you'll need to update the exclude property of the TypedocWebpackPlugin. It should be excluding this file. (Found in webpack.config.js).